stripe/src/main/java/com/stripe/android/model/WeChat.kt

```diff
@@ -7,7 +7,7 @@ import kotlinx.parcelize.Parcelize
  */
 @Parcelize
 data class WeChat internal constructor(
-    val statementDescriptor: String?,
+    val statementDescriptor: String? = null,
```

Comment:
What prompted this change?

Reply:
the intent response doesn't have "statement_descriptor" and "qr_code_url" fields